corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js

```diff
@@ -33,11 +33,12 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
             };
             if (menuResponse.type === "commands") {
                 menuListView = new MenuList.MenuListView(menuData);
+                FormplayerFrontend.regions.main.show(menuListView.render());
             }
             else if (menuResponse.type === "entities") {
                 menuListView = new MenuList.CaseListView(menuData);
+                FormplayerFrontend.regions.main.show(menuListView.render());
             }
-            FormplayerFrontend.regions.main.show(menuListView.render());
```

Comment:
not related but we do - the form response will also come through here which is not displayed in the menu region